packages/nimble-components/src/icon-base/tests/icons.react.tsx

```diff
@@ -0,0 +1,363 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable @typescript-eslint/naming-convention */
+import * as React from 'react';
```

Comment:
Don't forget to update the instructions for adding a new icon to mention also updating this file.

Reply:
We definitely should be modifying the icon build script instead https://github.com/ni/nimble/blob/main/packages/nimble-components/build/generate-icons/source/index.js

Reply:
Added, then removed, the instructions as the script creates the React wrappers for icons automatically now.